ui_framework/doc_site/src/services/routes/routes.js

```diff
@@ -160,6 +163,9 @@ const components = [{
   component: ModalExample,
   hasReact: true,
 }, {
+  name: 'Pager',
+  component: PagerExample,
```

Comment:
@Stacey-Gammon I just added this feature; with `hasReact: true`, a little React logo will show up next to the navigation item for this example.